docs/using/config/configuration.rst

```diff
@@ -134,6 +134,9 @@ The following parameters are optional when defining an application:
 
   This specifies the restart policy for this Application.
   There must be a ``name`` sub-key whose value may be one of: ``never``, ``always``, or ``on-failure``.
+  **The supplied policy is recorded but not not used.
```

Comment:
Maybe add link to issue for re-enabling this? Public issue tracker, may as well use that.